catalog/users/atma_io.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "atma_io",
+  "kind": "solutionprovider",
+  "name": "atma.io",
+  "email": "hello@atma.io",
+  "website": "https://www.atma.io/"
```

Comment:
You might want to add the optional field

```
"logo": "<url-to-your-logo>"
```

That ensures that in your logo is displayed in the collaborators page.

I am sorry that the possibility of adding the "logo" field was not explicit in the documentation. I have updated it meanwhile.